sdk/extensions/Azure.Extensions.AspNetCore.DataProtection.Blobs/src/ConfigureKeyManagementBlobClientOptions.cs

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Microsoft.AspNetCore.DataProtection.KeyManagement;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Options;
+
+#pragma warning disable AZC0001 //
```

Comment:
nit:  Please leave a short description of the rule being suppressed.

Reply:
Wasn't necessary - Builds without any warnings even without. Have removed